uk.co.compucorp.civicrm.hrcore/gulp/tasks/backstopjs.js

```diff
@@ -131,30 +131,33 @@ function runBackstopJS (command) {
 
   return getRolesAndIDs()
     .then(function (contactIdsByRoles) {
-      return new Promise(function (resolve) {
+      return new Promise(function (resolve, reject) {
         var isBackstopJSSuccessful;
 
         gulp.src(BACKSTOP_DIR_PATH + FILES.tpl)
           .pipe(file(destFile, tempFileContent(contactIdsByRoles)))
           .pipe(gulp.dest(BACKSTOP_DIR_PATH))
           .on('end', function () {
-            var promise = backstopjs(command, {
+            backstopjs(command, {
               configPath: BACKSTOP_DIR_PATH + destFile,
               filter: argv.filter
-            }).then(function () {
-              isBackstopJSSuccessful = true;
-            }).catch(_.noop).then(function () { // equivalent to .finally()
-              gulp
-                .src(BACKSTOP_DIR_PATH + destFile, { read: false })
-                .pipe(clean())
-                .pipe(notify({
-                  message: isBackstopJSSuccessful ? 'Successful' : 'Error',
-                  title: 'BackstopJS',
-                  sound: 'Beep'
-                }));
-            });
-
-            resolve(promise);
+            })
+              .then(function () {
+                isBackstopJSSuccessful = true;
+              })
+              .catch(_.noop).then(function () { // equivalent to .finally()
```

Comment:
I think `catch` block should be "hidden". I pay too much attention to it in the current code design.

```js
.then(function () {
}).catch(0)
.then(function () { // < I want to see the then block, not the catch block first

})
```

Reply:
That would be a very unusual syntax that we don't have anywhere else, so I don't think it would be a good idea.

The only two choices that i see are what is there now or 
```js
.then(function () {
})
.catch(_.noop)
.then(function () {
});
```
but given that the `catch` + `then` series is meant to be a substitute of just one command, it made sense to me to put them in one line "as one" rather than one after the other

Reply:
Please use the second one and not the `.catch(_.noop).then(function ()` since it is difficult to read.